app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java

```diff
@@ -700,11 +699,9 @@ private void onMaybeMuteChanged() {
             item.setTitle(player.isMuted() ? R.string.unmute : R.string.mute);
 
             //2) Icon change accordingly to current App Theme
-            TypedArray a = getTheme().obtainStyledAttributes(R.style.Theme_AppCompat, new int[]{R.attr.volume_off});
-            int attributeResourceId = a.getResourceId(0, 0);
-            Drawable drawableMuted = getResources().getDrawable(attributeResourceId);
-            Drawable drawableUnmuted = getResources().getDrawable(R.drawable.ic_volume_off_gray_24dp);
-            item.setIcon(player.isMuted() ? drawableMuted : drawableUnmuted);
+            item.setIcon(player.isMuted()
+                    ? getResources().getDrawable(ThemeHelper.resolveResourceIdFromAttr(getApplicationContext(), R.attr.ic_volume_off))
```

Comment:
I wasn't aware of ThemeHelper.resolveResourceIdFromAttr (and it seems like the one who first put the button, and the reviewers too),
you should rebase, because it [was changed yesterday](https://github.com/TeamNewPipe/NewPipe/pull/3252/files#), now both muted and unmuted use theme.

Reply:
Done, is it ok now?

Reply:
I haven't tested yet but the code seems ok

Reply:
I tested and it worked as expected. Though for now do not test this PR as it is not ready ;-)